api/desecapi/serializers.py

```diff
@@ -243,12 +244,15 @@ def get_validators(self):
 
     @staticmethod
     def validate_type(value):
-        if value in models.RRset.DEAD_TYPES:
-            raise serializers.ValidationError(f'The {value} RRset type is currently unsupported.')
-        if value in models.RRset.RESTRICTED_TYPES:
-            raise serializers.ValidationError(f'You cannot tinker with the {value} RRset.')
-        if value.startswith('TYPE'):
-            raise serializers.ValidationError('Generic type format is not supported.')
+        if value not in models.RR_SET_TYPES_MANAGEABLE:
+            # TODO see if SMIMEA can become a manageable type
+            # user cannot manage this type, let's try to tell her the reason
+            if value in models.RR_SET_TYPES_AUTOMATIC:
+                raise serializers.ValidationError(f'The {value} RRset(s) are managed by the API automatically.')
```

Comment:
`f'You cannot tinker with the {value} RRset. It is managed automatically.')`

Reply:
No other error message addresses the user personally

Reply:
So? :-)